portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,17 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addFunFact() {
+  // adjusted from greetings to facts about me!
+  // functionality is the same as addRandomGreeting(),
+  // so content in array is replaced while the basic functionality remains the same
+  const facts =
+      ['I play the cello!', '¡Soy peruana y americana!', 'I love Marvel!', '¡Me encanta dibujar!', 'I am from Seattle!', 'I love watching basketball!'];
 
   // Pick a random greeting.
```

Comment:
Change greeting to fact? Comments and code should be in sync.

Reply:
Greeting is now changed to fact. This js file should now be consistent!